api/lib/getJSON.js

```diff
@@ -138,13 +138,22 @@ const getSource = shabad => ({
   pageNo: shabad.PageNo,
 });
 
+/* eslint-disable no-param-reassign */
```

Comment:
👎 can we follow a more [functional programing](https://www.youtube.com/watch?v=e-5obm1G_FY) approach? I know we have tech debt already so if you want to wait for fixing all of it in one go, we can follow up.

Reply:
Let's start work on v3 this year and include typescript and some of that work. better integration of multisearch and graphql.